server/datastore/mysql/maintained_apps.go

```diff
@@ -128,6 +129,10 @@ WHERE NOT EXISTS (
 	)
 )`
 
+	if opt.MatchQuery != "" {
+		stmt += fmt.Sprintf(`AND (fla.name LIKE '%%%s%%')`, opt.MatchQuery)
```

Comment:
Careful! Formatting a string like this will lead to an sql injection. Try replacing it with a parameter `?`, and then add the query with the percent signs around it as an argument

Reply:
thanks updated this.